easybuild/easyconfigs/f/Fortran-YAML/Fortran-YAML-20210420-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "CMakeMake"
+
+name = 'Fortran-YAML'
+version = '20210420'
+local_commit = '2d203312f4ee057673e6d5db94542ae770813048'
+
+homepage = 'https://github.com/BoldingBruggeman/fortran-yaml'
+description = "Lightweight YAML parser written in object-oriented Fortran."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'BoldingBruggeman'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
```

Comment:
I prefer this for unversioned downloads
```
sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
```
that way the downloaded filename has a more clear relation to the naming of the easyconfig.